### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by mehulminat**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-hi.md [hunk 1]
```diff
@@ -22,3 +22,8 @@
 ### Javascript
 
 * [JavaScript Tutorial In Hindi](https://www.youtube.com/watch?v=hKB-YGF14SY) - Harry
+
```
Comment (SethiPandi) [inline free-courses-hi.md:25]: There should only be two line breaks between sections, not three.

**Review by SethiPandi — COMMENTED** [review]
Maybe read the contribution guide for clear guidance on formatting for this repository!

**Comment by eshellman** [comment]
please resolve conflicts.

**Review by mehulminat — COMMENTED** [review]
changes acc. to contribute.md

**Comment by eshellman** [comment]
still conflicts

**Comment by eshellman** [comment]
closing because added link points to a file sharing site.

**Pull request closed without merging**